### ISSUE TITLE
Can't delete old photos

### ISSUE THREAD

**Issue by duiveldoder** [kartaview/openstreetcam.org#314]
A while ago I was testing with Kartamap and uploaded a bunch of "trial" photos.

now I've learned how to do it much more accurate to get photos more accurate along existing OSM data.

So I deleted my trial-photos..

Although on my account it shows they are deleted, they remain on the map.

Of course as long as the 200 or 300 trial photos aren't removed, the over 10k and future 100k + photos I'm currently working at to get this whole city done .. well I won't upload a single of them as long as the old photos aren't removed.

If my request, that MY photos should be deleted isn't honored, then I won't upload a single more photo!

Very bad that Kartamap says they do NOT honor the international laws on photography and copyrights.

Good luck with your program though, without me then

**Comment by foobarna** [comment]
Hi @duiveldoder,
First of all, we are happy that you are looking into KartaView. Second, we are sad that you've experienced this bug. As a general overview, when a user deletes its photos, it will take a while until the processing pipelines honors the request - or, we might even got a bug. Not long ago we've completed some requests for some users to delete their data.

Please tell us the username and sequences with this problem. If you don't like to share that information here, please get in touch with us as per https://kartaview.org/privacy-policy#PrivacyPolicy10

Hope we see you soon!

**Comment by duiveldoder** [comment]
Hi Foobarna

https://kartaview.org/user/mrvinnie

just remove everything from my account - but not my account!!!

So I have a clean slate and can add whole sequences properly from here on.

My newly method is to walk down a street from start to end and then back.. every 0.5, 1, or 2 seconds I make a photo while walking.. and later home I align them properly based on what's on the photo and local knowledge where the objects I see are in reality and the corresponding OSM existing data I can get these images pretty accurately done - took me a lot of attempts at first but now got a very good workflow to go out and walk - for health and while improving help improving streetview coverage.. 2 birds with a single shot hehe

**Comment by miruna-craciun** [comment]
Hi Vincent, saw your email too, let's discuss here. 
I can see that you've embarked on quite the journey around Baguio - we're here to support that.
@foobarna will ensure that your KartaView profile will remain blank.
I'll get back to you on this ticket & via email when it's done. 
Thanks for raising this!

**Comment by duiveldoder** [comment]
Thank you for the quick response!

Basically the blue lines you can see here:
https://kartaview.org/map/@16.412164256654222,120.57845640604955,15z
are the ones that should be gone. The sequences I've deleted several days ago - but the blue lines representing them are still there.

____________________________________________________________________________________________________
While not waiting uploading my newest (manually corrected gps headers in the exif) uploads I contributed I noticed several issues around the Baguio Convention center on the Kartaview website (note the JOSM plugin shows everything just perfectly fine)

Looking at the blue lines here: https://kartaview.org/map/@16.403795988992627,120.60000940104226,17z
and compare them to JOSM Kartaview plugin (I'm going to assume you have JOSM with Kartaview plugin)

the JOSM Kartaview plugin shows the data 100% accurate as it should be. 

Following I'll post here the major "wrongs" by the kartaview website (1 post at a time) so it's easier to distinguish and review what could be optimized (if doable) hoping this would help you to optimize the KartaView web experience of users.

I'll make screenshots and hope I can send to each issue the JOSM-KartaView Plugin (this is perfect) and KartaView Web Experience (this could be improved)

**Comment by duiveldoder** [comment]
![issues 1 and 2](https://user-images.githubusercontent.com/15773876/112065722-1fd4cc80-8ba0-11eb-8e46-fc7c7dcebe42.png)

on the uploaded picture you can see issue 1:
it doesn't match to the end of the highway=residential (probably because the KartaView method wants it to connect to existing path - which it doesn't because it connects to another highway I uploaded afterwards - so instead of just following the road - it went through the trees, through the grass, through the footway, through the fence bordering the convention center parking lot, through the parking lot, through the road, to connect it to the side of that road I've been walking  just to make a connection - but that's wrong.. obviously.. it should just follow the road, not try to connect to anything - because what it should connect to, I uploaded an hour later (slow internet here)

So although 2 issues - they are related.. just pointing out here the 2 issues that are visible - are in fact a result of 1 and the same wrong assumption by the routine on the KartaView website.

Hope reporting this could help the dev team to catch such issues.

**Comment by duiveldoder** [comment]
![issue 3](https://user-images.githubusercontent.com/15773876/112067064-717e5680-8ba2-11eb-83a9-8076c9688c0b.png)

This would be issue 3. I marked with a red line to emphasize where the "blue track" should be on KartaView.

In fact, I haven't had the time yet to add this to OSM, but it's a footway I've covered along and outside the convention center.

It should NOT align at all with the existing highway inside the convention center fences - since it's not yet mapped.

It's starting point is at exactly the same spot as the previous 2 sequences.

I think for a case like this, it might be helpful for a contributor to "mark a sequence as - don't align with existing OSM data"

because it would be weird if anyone opening this sequence, to see the underlying map they're walking the parking area around the convention center and then instead see a footway along a fence on the outer side.

**Comment by duiveldoder** [comment]
![issue 4](https://user-images.githubusercontent.com/15773876/112068259-84922600-8ba4-11eb-91c0-3055141cc804.png)

This would be issue 4 and the last..
the other issues I mentioned happen a few times throughout my contribution around the convention center mainly because the area isn't accurately mapped, and my idea was to first upload everything and then use the images using the Kartaview plugin to correct everything around the convention center to be as accurate as possible.

But this issue 4 it's not just an "better not align at all OR let the user manually set start and end-points" but this one the routine being used doesn't even snap to the nearest OSM data - it simply jumpst here across residential area to a whole different road.

Although issues 1 to 3 are a result of not yet accurately have all data in OSM - this is really jumping from the right highway=residential to the complete wrong one. and you can see on JOSM, it's not like I am that far off from existing OSM LOL

**Comment by duiveldoder** [comment]
Well anyway - I see meanwhile the newly uploaded sequences have also already been removed.. so I'll wait now for the whole account to be "cleaned" LOL

meanwhile I have dozens of more sequences I need to adjust using JOSM properly and since it's beautiful weather outside going to add a few more roads today so I keep busy..

once the account is clean I can then start uploading the dozens of roads I've already covered so Baguio gets a nice start to be covered..

my goal is the main tourist areas like convention center - burnham park - sessions road - Camp John Hay - Wet Market and the area I'm living in Irisan..

And the target is to get that done before the end of April .. so if you can clean my account asap - I can every day upload 1 or 2 more streets hehe

**Comment by duiveldoder** [comment]
Follow up request

I see when I go into my account page [https://kartaview.org/user/mrvinnie](url) everything is clean now... basically back to empty account.

So I'm good to go to start uploading again right?

The only thing that's still left is the "empty sequences" which are visible on the map [https://kartaview.org/map/@16.40652175390528,120.58055071561353,15z](url)

since I'm the only contributor here - all these purple sequences are basically the empty ones.. which are now removed from my account but still online.

Is this map updated only once per month or why are these purple sequences still there?

**Comment by miruna-craciun** [comment]
@duiveldoder we appreciate your patience!
Indeed, your accout has been cleaned up.
The issue regarding deleted traces not disappearing from the map has been fixed. 
Let us know if there are other issues.
